### ISSUE TITLE
Quick edit gives error and loses category

### ISSUE THREAD

**Issue by hsq** [firesale/firesale#177]
master 1.0.5

Creating products and uploading images goes fine. But when I push the button 'quick edit' in the list of products, hit save, I get an internal error... 

ajax_quick_edit
/admin/firesale/products
POST
500
Internal Server Error
text/html
jquery.js:8102
Script

A Database Error Occurred
Error Number: 1146
Table 'database.dbname_default_firesale_products_firesale_categories' doesn't exist
INSERT INTO `dbname_default_firesale_products_firesale_categories` (`row_id`, `firesale_products_id`, `firesale_categories_id`) VALUES ('1', '15', '2')

Filename: /public_html/addons/siteref/modules/firesale/models/products_m.php
Line Number: 405

Solution:
change line 405:
                    $this->db->insert('default_firesale_products_firesale_categories', $data);

into

```
                $this->db->insert('firesale_products_firesale_categories', $data);
```

'default_' must be gone

Fixing this, solves it...

**Comment by ChristianGiupponi** [comment]
I think that this has been fixed in the 1.1 because I'm not able to reproduce your error

**Comment by jHoldroyd** [comment]
Fixed in both master and 1.1/develop. Thanks for reporting it, weird that it was left in there.
